zipline/src/engineMain/kotlin/app/cash/zipline/ZiplineManifest.kt

```diff
@@ -135,10 +137,18 @@ data class ZiplineManifest private constructor(
    */
   val signaturePayload: ByteString
     get() {
-      val signaturePayloadString = signaturePayload(encodeToString())
+      val signaturePayloadString = signaturePayload(encodeJson())
       return signaturePayloadString.encodeUtf8()
     }
 
+  fun encodeJson(): String {
```

Comment:
I’m okay with a public API for this.

Reply:
Definitely. I was going to suggest almost _exactly_ this on the other PR.